torchaudio/pipelines/_wav2vec2/impl.py

```diff
@@ -60,21 +77,29 @@ def _get_state_dict(self, dl_kwargs):
         state_dict = load_state_dict_from_url(url, **dl_kwargs)
         return state_dict
 
-    def get_model(self, *, dl_kwargs=None) -> Wav2Vec2Model:
+    def get_model(self, *, dl_kwargs=None) -> Module:
         """Construct the model and load the pretrained weight.
 
         The weight file is downloaded from the internet and cached with
         :func:`torch.hub.load_state_dict_from_url`
 
         Args:
             dl_kwargs (dictionary of keyword arguments): Passed to :func:`torch.hub.load_state_dict_from_url`.
+
+        Returns:
+            Variation of :py:class:`~torchaudio.models.Wav2Vec2`.
+            If self.params_["normalize_waveform"] is True, Wav2Vec2 is wrapped with _Wav2Vec2 to apply
+            an additional layer normalization at the input
```

Comment:
```suggestion
```

`self.params_` is implementation detail so it should not mentioned in public-facing documentation. Rather, can you describe in plain English for what models the extra normalization is applied?